### ISSUE TITLE
Image not resizing when cache disabled (dev-master)

### ISSUE THREAD

**Issue by BD-Rene** [thephpleague/glide#54]
When using the latest cache disabling stuff in dev-master, it seems that image resizing no longer works correctly. Haven't tried any other image manipulation, but something like this:

```
$server = League\Glide\ServerFactory::create([
'source' => dirname($_SERVER['SCRIPT_FILENAME']) . '/uploads/resources',
'cache' => false,
]);

$server->outputImage(thumbnail.jpg', [
'w'    => '200,
'fit'  => 'crop',
'crop' => 'center'
]);
```

... basically results in it outputting the original image without resizing or cropping.

Turning cache back on resolves this issue.

**Comment by kevin-thackorie** [comment]
Confirmed. I am able to reproduce this issue and am investigating it now.

**Comment by reinink** [comment]
This has been "fixed" in [0.3.5](https://github.com/thephpleague/glide/releases/tag/0.3.5), as the cache disabling now uses `sys_get_temp_dir()` when no cache is set.

However, we aware that caching disabling will be completely removed in the next version of Glide, in favour of a new in-memory adapter that handles this much nicer. See [here](https://github.com/thephpleague/glide/pull/56#issuecomment-112082800) for more information on that.
